### PR TITLE
Add dev watch build

### PR DESCRIPTION
**Pull request by dan-lee**
- Added `pnpm dev` script to watch and build
- Only minify when it's _not_ a dev build
- Move `example` to pnpm
- Got breakpoints working again:

https://user-images.githubusercontent.com/571589/232288373-11fee4f0-8781-400f-8ab1-2a7487385868.mp4

_Resolves: #21_

### CODE REVIEW

**Comment by JoviDeCroock** [comment]
Updated to a monorepo in #33 as this PR broke changesets and publishing in general

File: scripts/launch-debug.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env zsh
```
Comment (JoviDeCroock) [inline scripts/launch-debug.sh:1]: Probably safer as not everyone uses zsh

```suggestion
#!/usr/bin/env sh
```

**Comment by dan-lee** [comment]
Rebased!

**Comment by JoviDeCroock** [comment]
@dan-lee awesome, you okay with me merging this, working well?

**Review by JoviDeCroock — APPROVED** [review]

**Comment by dan-lee** [comment]
Yeah of, course happy to see it merged! I rechecked breakpoints, works fine :)

**Comment by dan-lee** [comment]
I could re-record the breakpoint video for the README. Might help lower the hurdle to contributing a bit, wdyt?

**Pull request merged**